CONTRIBUTING.md

```diff
@@ -16,6 +16,7 @@ This section describes the main steps required before you can add your code and
 * Some container engine: either `docker` or `podman`.
 * go: v1.20
 * Python 3.9 and pipenv.
+* Prometheus
```

Comment:
This list contains the stuff that should be installed on the developer's machine. I don't think they need to have Prometheus installed.